src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import categoriesReducer from './categories/categories';
+
+const reducer = combineReducers({
+  categories: categoriesReducer,
+});
```

Comment:
- [ ] KIndly import the `books` reducer and add it using `combineReducers` as you did with the `categories` reducer.

Reply:
Thanks I have fixed it